### DISCUSSION TITLE
Lesson 2 Clone.......sort of

### DISCUSSION THREAD

**Discussion by yeahChibyke** [Cyfrin/foundry-full-course-cu#350] [Show and tell]
I finished Lesson 2 yesterday, and I decided to write my own smart contract -MyFrenGroup- using what I learnt from @PatrickAlphaC teaching.
Plus, I deployed it using to the Sepolia network using my Metamask wallet. 
![MyFrenGroup](https://github.com/Cyfrin/foundry-full-course-f23/assets/56419077/ac19ee75-e554-4b9e-b24a-590ed8c48eda)

**Comment by ghost** [comment]
Pretty neat 👍🏻.
